index.js

```diff
@@ -44,22 +44,24 @@ const quantifierTable = {
   betweenLazy: metadata => `{${metadata[0]},${metadata[1]}}?`,
 }
 
+const applySubexpressionDefaults = expr => {
+  const out = { ...expr };
+  out.namespace = ('namespace' in out) ? out.namespace : '';
+  out.ignoreFlags = ('ignoreFlags' in out) ? out.ignoreFlags : false;
+  out.ignoreStartAndEnd = ('ignoreStartAndEnd' in out) ? out.ignoreStartAndEnd : false;
```

Comment:
I wonder if these should default to `true`.

To me, flags are a global concept which sort of clashes with subexpressions.

Reply:
Yeah I think I agree actually. Explicitly granting the permission seems like it would cause less unexpected problems for people in the end.